pde_superresolution/integrate.py

```diff
@@ -110,6 +112,24 @@ def __call__(self, t: float, y: np.ndarray) -> np.ndarray:
     time_derivative = self.equation.equation_of_motion(y, space_derivatives)
     return self.equation.finalize_time_derivative(t, time_derivative)
 
+class WENODifferentiator(Differentiator):
+  """Calculate derivatives using a 5th order WENO.py method."""
+
+  def __init__(self, equation: equations.Equation, k: int = 3):
+    assert hasattr(equation, 'flux')
+    self.equation = equation
+    self.weno = WENO(k=k, dx=equation.grid.reference_dx)
+    self.alpha = 1.0  # For Lax-Friedrich monotone flux
```

Comment:
where does this number come from? the paper suggests it should be the max value of first derivative (which is likely a bit larger than this, maybe closer to 10 just judging by eye).

Reply:
just a default. I fiddled with different values, which do indeed give different results.